test/GraknConsoleIT.java

```diff
@@ -432,6 +432,21 @@ public void when_ErrorOccurs_expect_noStackTrace() {
         assertThat(response.err(), not(containsString(".java")));
     }
 
+    @Test
+    public void when_ListKeyspaces_KeyspacesAreListed() throws Exception {
+        runConsoleSession("", "-k", "keyspace_a");
+        runConsoleSession("", "-k", "keyspace_b");
+        runConsoleSession("", "-k", "keyspace_c");
```

Comment:
where did you declare "-k" as  the shorthand of the argument?

Reply:
Unrelated - these is just opening three console sessions connected to different keyspaces so the test will list multiple keyspaces. Will add comment